src/App.tsx

```diff
@@ -79,12 +44,12 @@ export const App: React.FC = () => {
             <div className="block">
               <TodoFilter
                 handleFilterTodos={handleFilterTodos}
-                filterTodos={filterTodos}
+                filterTodos={todosFilter}
               />
             </div>
 
             <div className="block">
-              {todos.length > 0 ? (
+              {isLoading ? (
```

Comment:
The naming is incorrect here.
I suggest:
- either to rename this constant to `isLoaded` (here and in the hook)
- or to switch to opposite values for it in the hook (set true instead of false and vice versa) and rewrite conditional here (`isLoading ? <Loader /> : <TodoList ... />`)